deltaspike/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-jvm</artifactId>
+        <version>4.3.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cucumber-deltaspike</artifactId>
+    <packaging>jar</packaging>
+    <name>Cucumber-JVM: Deltaspike</name>
+
+    <properties>
+        <deltaspike.version>1.9.0</deltaspike.version>
+        <cdi-api.version>2.0.SP1</cdi-api.version>
+        <jaxb-api.version>2.3.1</jaxb-api.version>
+
+        <weld-se.version>2.4.4.Final</weld-se.version>
+        <openejb-core.version>8.0.0-M1</openejb-core.version>
+        <owb.version>2.0.10</owb.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cucumber-java</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.deltaspike.cdictrl</groupId>
+            <artifactId>deltaspike-cdictrl-api</artifactId>
+            <version>${deltaspike.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.enterprise</groupId>
+            <artifactId>cdi-api</artifactId>
+            <version>${cdi-api.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cucumber-junit</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>deltaspike-weld</id>
```

Comment:
Why the different profiles? It would appear they all go unused.

Reply:
to show that you can use weld, openejb and openwebbeans with this module? you're right only weld will be used at default. But I'm unsure how to test the other cdi-containers here or to show that the will work? At a later stage we can replace the profiles with to "old" modules?

Reply:
It's nice as a demonstration. However this is an open source project which means that any sort of manual test will soon fall by the wayside. Might be better to remove it up all together.

Reply:
okay, what do you think about replace profiles with examples for the different containers? Than I can referent the doc also to this?

Reply:
Just the default profile should be fine I think. Delta Spike is a facade for the different libraries. You can put the examples in the docs.